varcode/effect_ordering.py

```diff
@@ -52,10 +52,18 @@ def top_priority_transcript_effect(effects):
         elif priority == best_priority:
             best_effects.append(effect)
 
-    def effect_cds_len(effect):
-        """
-        Returns length of coding sequence of transcript associated with effect
-        """
-        return len(effect.transcript.coding_sequence)
-
-    return max(best_effects, key=effect_cds_len)
\ No newline at end of file
+    if any(effect.transcript.complete for effect in best_effects):
```

Comment:
I don't really understand the context here, but why not apply the same sorting to both (first by CDS length, then by transcript length), vs. filtering in one case and not in the other?

(Probably a silly question. I'm just not clear on the context.)